### PR TITLE
Feature/ipa production plant improvement

### PR DESCRIPTION
**Pull request by ipa-srd-rd**
In this PR the new environment ipa-production-plant has been updated. New Object locations, Reflector Marker positions and maps were added to this environment. 

This PR is related to  https://github.com/ipa320/cob_simulation/pull/110

### CODE REVIEW

File: cob_default_env_config/upload_object_locations.launch [hunk 1]
```diff
@@ -5,7 +5,6 @@
 
 	<!-- send objects locations to parameter server -->
 	<rosparam command="delete" param="/objects"/>
-	<rosparam command="load" ns="objects" file="$(find cob_default_env_config)/$(arg robot_env)/object_locations.yaml"/>
-
+        <rosparam command="load" ns="objects" file="$(find cob_default_env_config)/$(arg robot_env)/object_locations.yaml"/>
```
Comment (floweisshardt) [inline cob_default_env_config/upload_object_locations.launch:8]: please revert tabs vs spaces

**Comment by floweisshardt** [comment]
same comment as for https://github.com/ipa320/cob_simulation/pull/110#discussion_r70616878. Can you please explain why you need the "room" concept?

**Comment by mgruhler** [comment]
Do not merge: included in #109

**Comment by ipa-srd-rd** [comment]
The functionality of this PR is included in #109 and will therefore be closed.

**Pull request closed without merging**